types/winattr/winattr-tests.ts

```diff
@@ -0,0 +1,39 @@
+import winattr from "winattr";
```

Comment:
since this is a commonjs module, you should use Typescript's commonjs syntax `import winattr = require('winattr')`

Reply:
Changes made